/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:40896'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:33488'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:40296'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:41935'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:43921'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:43315'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:42985'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:46654'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:43392'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:37775'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:35572'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:45050'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:39352'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:40409'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:46007'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:36586'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:39181'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:33873'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:39554'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:36123'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:44219'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:41342'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:41828'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:44369'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:33927'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:43934'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:39689'
distributed.nanny - INFO -         Start Nanny at: 'tcp://172.30.8.63:46529'
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:36107
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:44283
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:46332
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:40869
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:36107
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:33826
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:35673
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:44283
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:46308
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:46332
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:38173
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:42088
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:43832
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:40869
distributed.worker - INFO -          dashboard at:          172.30.8.63:34507
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:43499
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:46227
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:36225
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:33826
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:35673
distributed.worker - INFO -          dashboard at:          172.30.8.63:38829
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:46308
distributed.worker - INFO -          dashboard at:          172.30.8.63:43370
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:38173
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:32849
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:42088
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:43832
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -          dashboard at:          172.30.8.63:40715
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:43499
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:46227
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:36225
distributed.worker - INFO -          dashboard at:          172.30.8.63:34776
distributed.worker - INFO -          dashboard at:          172.30.8.63:32793
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -          dashboard at:          172.30.8.63:38978
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -          dashboard at:          172.30.8.63:44967
distributed.worker - INFO -          dashboard at:          172.30.8.63:40967
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:32849
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:          172.30.8.63:37812
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:33561
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:43104
distributed.worker - INFO -          dashboard at:          172.30.8.63:34184
distributed.worker - INFO -          dashboard at:          172.30.8.63:44395
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -          dashboard at:          172.30.8.63:35866
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/contextlib.py:119: UserWarning: Creating scratch directories is taking a surprisingly long time. This is often due to running workers on a network file system. Consider specifying a local-directory to point workers to write scratch data to a local disk.
  next(self.gen)
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:38744
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:36896
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:39521
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:32795
distributed.worker - INFO -          dashboard at:          172.30.8.63:40335
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:38659
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:44896
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:33561
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:43104
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:46304
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:44281
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:38983
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:40999
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:41619
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:38744
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:36896
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:39521
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:44896
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:32795
distributed.worker - INFO -          dashboard at:          172.30.8.63:33835
distributed.worker - INFO -          dashboard at:          172.30.8.63:33636
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:46304
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:38983
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:44281
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:41619
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:40999
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:          172.30.8.63:38370
distributed.worker - INFO -          dashboard at:          172.30.8.63:41905
distributed.worker - INFO -          dashboard at:          172.30.8.63:39798
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ojogofem
distributed.worker - INFO -          dashboard at:          172.30.8.63:42395
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/utils.py:136: RuntimeWarning: Couldn't detect a suitable IP address for reaching '8.8.8.8', defaulting to '127.0.0.1': [Errno 101] Network is unreachable
  RuntimeWarning,
distributed.worker - INFO -       Start worker at:    tcp://172.30.8.63:38115
distributed.worker - INFO -          dashboard at:          172.30.8.63:40210
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-05kkmhb7
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:38659
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -          dashboard at:          172.30.8.63:39226
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-9jyi766x
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-wqw8q4yc
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          dashboard at:          172.30.8.63:33713
distributed.worker - INFO -          dashboard at:          172.30.8.63:33824
distributed.worker - INFO -          dashboard at:          172.30.8.63:37177
distributed.worker - INFO -          dashboard at:          172.30.8.63:46594
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-n9yzuchw
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          Listening to:    tcp://172.30.8.63:38115
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-y_lh1ty8
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-vnro3s8t
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-yj9uj8l8
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -          dashboard at:          172.30.8.63:42663
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-0oavo_7j
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-5w3o8a3c
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-julqjafq
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -          dashboard at:          172.30.8.63:43302
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-ilykxrhk
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-nh1g51ob
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-9nh86vv7
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - Waiting to connect to:   tcp://172.30.100.1:46491
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-fya74mhc
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-q_p6bfy2
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-595oukpv
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-go_ml3nv
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-1w0ar6u6
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-4wcfm7ub
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -               Threads:                          1
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-40uwhhoh
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-duzif9qa
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-vblffzne
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-mlo7_2rh
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-e84gg7rn
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-379drk__
distributed.worker - INFO -                Memory:                    4.29 GB
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-nwbk_3uj
distributed.worker - INFO -       Local Directory: /scratch/cnt0024/hmg2840/albert7a/daskjobqueue/worker-buehukhv
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.worker - INFO -         Registered to:   tcp://172.30.100.1:46491
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.worker - INFO - -------------------------------------------------
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
distributed.core - INFO - Starting established connection
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34162 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34190 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34164 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34552 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34696 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34504 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34700 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34692 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34716 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34690 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34714 remote=tcp://172.30.100.1:46491>
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
tornado.application - ERROR - Exception in callback functools.partial(<bound method IOLoop._discard_future_result of <tornado.platform.asyncio.AsyncIOLoop object at 0x2ada04a3a160>>, <Future finished exception=OSError("Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time")>)
Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 218, in connect
    quiet_exceptions=EnvironmentError,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
tornado.util.TimeoutError: Timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 743, in _run_callback
    ret = callback()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/ioloop.py", line 767, in _discard_future_result
    future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/worker.py", line 791, in heartbeat
    address=self.contact_address, now=time(), metrics=self.get_metrics()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 738, in send_recv_from_rpc
    comm = yield self.pool.connect(self.addr)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/core.py", line 866, in connect
    connection_args=self.connection_args,
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 729, in run
    value = future.result()
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/tornado/gen.py", line 736, in run
    yielded = self.gen.throw(*exc_info)  # type: ignore
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 230, in connect
    _raise(error)
  File "/scratch/cnt0024/hmg2840/albert7a/anaconda3/lib/python3.7/site-packages/distributed/comm/core.py", line 207, in _raise
    raise IOError(msg)
OSError: Timed out trying to connect to 'tcp://172.30.100.1:46491' after 10 s: connect() didn't finish in time
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34958 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34940 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34950 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34858 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34952 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34942 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34956 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34194 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34694 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34502 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34718 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34526 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34742 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34550 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34598 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34622 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34670 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34766 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34382 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34406 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34430 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34454 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34574 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35310 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34188 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34724 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34748 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34556 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34676 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34412 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34460 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34580 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34628 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35044 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34900 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34924 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34972 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35304 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35184 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35308 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35300 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35180 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34182 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34720 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34530 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34746 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34554 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34600 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34672 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34768 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34386 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34408 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34432 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34456 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34576 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34626 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34944 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35040 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34896 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34922 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34968 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34192 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34500 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34524 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34740 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34548 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34596 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34620 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34668 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34764 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34380 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34404 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34428 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34450 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34812 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34572 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35036 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34892 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34914 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34964 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35302 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35182 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34186 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34212 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34736 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34544 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34688 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34712 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34760 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34376 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34808 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34448 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34472 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34810 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34592 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34616 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34866 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35056 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34888 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34912 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34936 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34960 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35130 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34498 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34522 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34738 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34546 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34594 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34618 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34666 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34762 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34378 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34402 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34426 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34452 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34814 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34570 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34938 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35034 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34870 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34890 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34916 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34962 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34730 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34538 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34754 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34562 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34802 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34706 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34418 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34442 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34466 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34610 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34634 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34860 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35050 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34906 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34930 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34954 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34185 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34728 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34536 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34752 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34560 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34800 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34608 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34680 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34704 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34416 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34440 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34464 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34584 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34632 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35048 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34880 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34904 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:34928 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35410 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35218 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35434 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35242 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35266 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35290 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35314 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35386 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35124 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35134 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35170 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35194 remote=tcp://172.30.100.1:46491>
distributed.comm.tcp - WARNING - Closing dangling stream in <TCP  local=tcp://172.30.8.63:35338 remote=tcp://172.30.100.1:46491>
slurmstepd: error: *** JOB 7329224 ON n2502 CANCELLED AT 2019-09-04T11:01:43 DUE TO TIME LIMIT ***
